pkg/reconcile/reconcilers.go

```diff
@@ -242,6 +242,24 @@ func (r *defaultReconciler) createSchemaJob(ctx context.Context, schemaJob *v1ba
 	schemaJob = newSchemaJob(reaper, cassdc.GetDatacenterServiceName())
 	key := types.NamespacedName{Namespace: schemaJob.Namespace, Name: schemaJob.Name}
 
+	if len(reaper.Spec.ServerConfig.CassandraBackend.AuthProvider.SecretRef) > 0 {
+		secret, err := r.getSecret(types.NamespacedName{Namespace: reaper.Namespace, Name: reaper.Spec.ServerConfig.CassandraBackend.AuthProvider.SecretRef})
+		if err != nil {
+			req.Logger.Error(err, "failed to get Cassandra authentication secret", "deployment", key)
```

Comment:
Can we fix this log statement? It has a key/value of `deployment` and the value is schema job key. Can you define a a `secretKey` var and change the logging statement to something like:

```go
req.Logger.Error(err, "failed to get Cassandra authentication secret", "cassandraUserSecret", secretKey)
```